.github/workflows/windows.yml

```diff
@@ -35,21 +35,22 @@ jobs:
         with:
           location: C:\msys2
           update: true
+          msystem: MINGW64
           install: |
             gzip
```

Comment:
gzip can also be removed btw, it's already part of msys2 [base](https://packages.msys2.org/package/base?repo=msys&variant=x86_64) install...